torch/_ops.py

```diff
@@ -782,6 +782,20 @@ def tags(self):
     # TODO: add more methods to expose information about input and output arguments
 
 
+# TorchBindOpOverload are those custom ops which have at least one overload's
+# schema consists of torch.ScriptObject (i.e. custom class) input.
+# TorchBindOpOverload will skip C++ dispatcher and purely dispatched in python
+# when its inputs contain FakeScriptObject.
+class TorchBindOpOverload(OpOverload):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._is_torchbind_op = True
```

Comment:
Why do you need a separate TorchBindOpOverload? `_is_torchbind_op` on an OpOverload isn't sufficient?

Reply:
We also need to add a new `__call__` and `__init__` logic in OpOverload later in the stack so I think it might be cleaner to just create a new subclass and put all its logic together. The alternative is to write a few `if _is_torchbind_op` checks, which seems a bit more messy.